src/gui/DatabaseWidget.cpp

```diff
@@ -1671,6 +1671,8 @@ bool DatabaseWidget::save()
     m_entryView->setDisabled(false);
     m_groupView->setDisabled(false);
 
+    m_entryView->setFocus();
```

Comment:
This is not the right thing to do because the entryview may not have had focus prior to saving. You should store the focus widget before disabling (`auto focusWidget = qApp->focusWidget()`) then restore the focus when enabled again `if (focusWidget) { focusWidget->setFocus(); }`